pyproject.toml

```
@@ -0,0 +1,66 @@
+[build-system]
+requires = ["setuptools", "wheel"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "statick-tooling"
+authors = [{name = "NIWC Pacific"}]
+description="Statick analysis plugins for Tooling files."
+version = "0.2.0"
+readme = "README.md"
+requires-python = ">=3.9"
+license = {text = "CC0-1.0"}
+classifiers = [
+    "License :: CC0 1.0 Universal (CC0 1.0) Public Domain Dedication",
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3.11",
+    "Programming Language :: Python :: 3.12",
+    "Programming Language :: Python :: 3.13",
+    "Topic :: Software Development :: Quality Assurance",
+    "Topic :: Software Development :: Testing",
+    "Typing :: Typed",
+]
+
+dependencies = [
+    "importlib_metadata",
+    "statick@git+https://github.com/tdenewiler/statick@stdlib-plugins",
+    "types-docutils",
+]
+
+[tool.setuptools.package-data]
+statick_tool = [
+    "rsc/*",
+    "rsc/.*",
+]
+
+[project.entry-points."statick_tool.plugins.discovery"]
+dockerfile = "statick_tool.plugins.discovery.dockerfile:DockerfileDiscoveryPlugin"
+
+[project.entry-points."statick_tool.plugins.tool"]
+dockerfile-lint = "statick_tool.plugins.tool.dockerfile_lint:DockerfileULintToolPlugin"
+dockerfilelint = "statick_tool.plugins.tool.dockerfilelint:DockerfileLintToolPlugin"
+hadolint = "statick_tool.plugins.tool.hadolint:HadolintToolPlugin"
+
+[project.urls]
+"Homepage" = "https://github.com/sscpac/statick-tooling"
+"Bug Tracker" = "https://github.com/sscpac/statick-tooling/issues"
+
+[project.optional-dependencies]
+test = [
+    "coverage",
+    "mock",
+    "pytest",
+    "pytest-cov",
+    "statick-md@git+https://github.com/tdenewiler/statick-md@stdlib-plugins",
```

```suggestion
    "statick-md>=1.0.0"
```
once that PR is merged and we have a new release.